Microsoft.Toolkit.Uwp.Input.GazeInteraction/OneEuroFilter.h

```diff
@@ -0,0 +1,65 @@
+//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include "IGazeFilter.h"
+
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+const float ONEEUROFILTER_DEFAULT_BETA = 5.0f;
+const float ONEEUROFILTER_DEFAULT_CUTOFF = 0.1f;
+const float ONEEUROFILTER_DEFAULT_VELOCITY_CUTOFF = 1.0f;
+
+public ref class LowpassFilter sealed
+{
+public:
+    LowpassFilter()
+    {
+        Previous = Point(0, 0);
+    }
+
+    LowpassFilter(Point initial)
+    {
+        Previous = initial;
+    }
+
+    property Point Previous;
```

Comment:
Should this be public set?

Reply:
The filter classes have been marked as private now.

Reply:
@harishsk It looks like `NullFilter` is still public? /cc @nmetulev